index.d.ts

```diff
@@ -1,8 +1,8 @@
 /**
- * An *action* is a plain object that represents an intention to change the
+ * An *action* is a plain object that represents an intention to change
  * state. Actions are the only way to get data into the store. Any data,
  * whether from UI events, network callbacks, or other sources such as
- * WebSockets needs to eventually be dispatched as actions.
+ * WebSockets need to eventually be dispatched as actions.
```

Comment:
To clarify, `needs` corresponds to `data` before. I think `needs` is correct here.